browser/webpack.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
```

Comment:
I'd also recommend renaming this file as webpack.config.js, as it's a conf file, not a webpacking module.